src/exercise/02.html

```diff
@@ -3,26 +3,23 @@
 
 <body>
   <div id="root"></div>
-
-  <!-- 💣 remove this line
   <script src="https://unpkg.com/react@17.0.0/umd/react.development.js"></script>
   <script src="https://unpkg.com/react-dom@17.0.0/umd/react-dom.development.js"></script>
-  💣 remove this line -->
 
   <script type="module">
-    const rootElement = document.getElementById('root')
-
-    // You're going to re-implement this code using React!
-    // 💣 So go ahead and delete this implementation (or comment it out for now)
-    // These three lines are similar to React.createElement
-    const element = document.createElement('div')
-    element.textContent = 'Hello World' // 💰 in React, you set this with the "children" prop
-    element.className = 'container' // 💰 in React, this is also called the "className" prop
-    // This is similar to ReactDOM.render
-    rootElement.append(element)
+    const rootElement = document.getElementById('root');
+    const reactElement = React.createElement('div', {
+      // Define some props for the container.
+      className: 'container',
+      children: [
+        // Toss some little guys in there.
+        React.createElement('span', null, 'Hello'),
+        '\n',
```

Comment:
Twist! First `\n` I've seen in these :)

Others have used `' '`. I didn't put any middle child.

Weirdly, all three create the same thing in the elements view:

![image](https://user-images.githubusercontent.com/7775971/123294527-ef743580-d4e2-11eb-8613-60d038d6127c.png)

But only the implementations with the middle `' '` or `'\n'` child have a space between the words in the rendered page.

![image](https://user-images.githubusercontent.com/7775971/123294659-0c106d80-d4e3-11eb-86cb-4e02c774199e.png)

Reply:
I _think_ that when react renders to the actual DOM, as it ultimately must do, it ultimately outputs something like a string for the browser to interpret, because this behaves more like what you'd see in "View Source" than in element view. No real evidence for that beyond the fact that if you don't force whitespace between the elements it acts the same a if they were rendered server-side (no space between the words), but that's the mental model I'm going to use for now 😄